content/10.blackbox.md

```diff
@@ -1 +1,6 @@
-## Tip 8: Do not necessarily consider a DL model as a black box {#blackbox}
+#Rule 8: Do not necessarily consider a DL model as a black box
+In ML, interpretability refers to the study of the discriminative features used for classification or regression task. In general, DL models need little data engineering and have a very large number of parameters (**TODO** link it somewhere else in the manuscript). Thus, DL models are relatively hard to interpret when compared to other ML models, such as random forests.
+
+However, methods such as Autoencoder (AE, **TODO** check if autoencoder is mentioned elsewhere) are DL strategies that can be used for model interpretation. AE is a family of unsupervised methods that aim to learn a new (encoded) representation and minimize the error between the new representation and the input data. Tan et al., [@doi:10.1142/9789814644730_0014] has uses denoising autoencoders to extract useful genomics features a from breast cancer dataset. Another example of  model interpretation is the Deep Bind [@doi:10.1038/nbt.3300], which predicts DNA and RNA binding sequences potentially involved in regulatory processes. Deep bind used mutation maps to understand the impact of genetic variants of these binding sites ... **TODO** 
+
+Unfortunelly, the literature on interpretable DL methods is extremily limited. **TODO** what should we suggest here?
```

Comment:
I don't really agree it is extremely limited.  The deep review described many different approaches for model interpretation (https://greenelab.github.io/deep-review/#interpretation) and there has only been more research in this area since then.

Reply:
I agree with @agitter here. Also, all methods that apply to conventional ML techniques (like feature importance estimation, permutation tests, etc.) apply to DL as well. I find DL models even more interpretable than say kernel SVMs or Random forests, as you can backtrace the gradient to establish input output relationships and e.g,. in convnets you can visualize the patterns extracted via the intermediate layers etc.

Along with that, a few typo suggestions:

> has uses denoising autoencoders

=> used denoising autoencoders

> extract useful genomics features

=> summarize key features from genomic datasets

> of  model interpretation is the Deep Bind 

> Another example of  model interpretation is the Deep Bind

=> Another example of extracting biological insights from deep learning models is provided by the DeepBind database

> Unfortunelly

=> Unfortunately

> extremily

extremely

...